roll20harn.css

```diff
@@ -395,6 +400,7 @@ input[type=number]::-webkit-inner-spin-button {
 /* ROLL TEMPLATES */
 .sheet-rolltemplate-check-lt table,
 .sheet-rolltemplate-rollcheck table {
+    background-image: url("https://jpmiii.com/bg.gif");
```

Comment:
what you should do here is put the image in source control, then link to the raw file here. That way the file is a) highly available b) people can see what it is when they checkout the repo c) not reliant on you always hosting it.